packages/components/src/Select.ts

```diff
@@ -0,0 +1,206 @@
+import _ from 'lodash/fp';
+import { div, h } from 'react-hyperscript-helpers';
+import RSelect, {
+  components as RSelectComponents,
+  GroupBase as RSelectGroupBase,
+  MultiValue as RSelectMultiValue,
+  Props as RSelectProps,
+  PropsValue as RSelectPropsValue,
+  SingleValue as RSelectSingleValue,
+} from 'react-select';
+
+import { useUniqueId } from './hooks/useUniqueId';
+import { icon } from './icon';
+import { useThemeFromContext } from './theme';
+
+let colors: any;
```

Comment:
Can we find a better way than a global variable to pass the colors from the theme context through to React Select's theme?